pynn_spinnaker/spinnaker/runtime/neuron_processor/neuron_dynamics_models/if.h

```diff
@@ -73,15 +70,14 @@ class IFCurr
   //-----------------------------------------------------------------------------
   // Static methods
   //-----------------------------------------------------------------------------
-  static inline bool Update(MutableState &mutableState, const ImmutableState &immutableState,
-                            S1615 excInput, S1615 inhInput, S1615 extCurrent)
+  static S1615 Update(MutableState &mutableState, const ImmutableState &immutableState,
+                      S1615 inputCurrent)
```

Comment:
Not sure that I would merge these currents prior to this point.  Keeping them separate allows the possibility of having a merged-in-conductance model for example (we would have to provide a Raw input type which would look very like the Curr, and a Passive synapse type)